governator-test/src/main/java/com/netflix/governator/guice/test/mocks/MockHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.netflix.governator.guice.test.mocks;
+
+
+/**
+ * Abstraction for creating/resetting Mocks and Spies. Must have a default constructor.
+ */
+public interface MockHandler {
```

Comment:
Handler might not be the right noun here.  RequestHandler is something that processes api requests, but this MockHandler creates and resets mocks.  Should this be a Factory or Builder?